de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/45-de.metas.pricing/5490520_gh3835_NoPriceConditionsColor_ID_SysConfig.sql

```diff
@@ -0,0 +1,6 @@
+-- 2018-04-12T16:28:25.005
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,Description,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541204,'S',TO_TIMESTAMP('2018-04-12 16:28:24','YYYY-MM-DD HH24:MI:SS'),100,'The value will contain the  AD_Color_ID that will be set in C_OrderLine.NoPriceConditionsColor_ID if the C_OrderLine.M_DiscountSchemaBreak_ID is null.
```

Comment:
pls also fix the description. It's the AD_Color.Name

Reply:
done